scalafix-tests/unit/src/test/scala-target2/scalafix/tests/cli/BaseCliSuite.scala

```diff
@@ -159,10 +159,16 @@ trait BaseCliSuite extends AnyFunSuite with DiffAssertions {
     )
   }
 
+  // The defaults of this helper expect to find test input which is currently
```

Comment:
only one user of this helper is actively relying on defaults and fails on Scala 3 targets, but I moved all of them for consistency